types/process.d.ts

```diff
@@ -183,5 +183,181 @@ declare module "process" {
      * ```
      */
     exit(code?: number | string | null | undefined): never;
+
+    /**
+     * The `process.getgid()` method returns the numerical group identity of the
+     * process. (See [`getgid(2)`](http://man7.org/linux/man-pages/man2/getgid.2.html).)
+     *
+     * ```js
+     * import process from 'node:process';
+     *
+     * if (process.getgid) {
+     *   console.log(`Current gid: ${process.getgid()}`);
+     * }
+     * ```
+     *
+     * This function is only available on POSIX platforms (i.e. not Windows or
+     * Android).
+     * @since v0.1.31
+     */
+    getgid?: () => number
+    /**
+     * The `process.setgid()` method sets the group identity of the process. (See [`setgid(2)`](http://man7.org/linux/man-pages/man2/setgid.2.html).) The `id` can be passed as either a
+     * numeric ID or a group name
+     * string. If a group name is specified, this method blocks while resolving the
+     * associated numeric ID.
+     *
+     * ```js
+     * import process from 'node:process';
```

Comment:
Lets remove node references

Reply:
We should also think about how to maintain the type in a simple way. Copying the type declaration of node and then removing `node:` doesn't seem to be a good idea.